### PR TITLE
Make lizard-specific sprites for masks and the CT helmet

### PR DESCRIPTION
**Pull request by Dutch-VanDerLinde**
Pictures:

image
image
image

### CODE REVIEW

**Comment by Hyenh** [comment]
Will you also do the rebreather and medical gas mask?

**Comment by Dutch-VanDerLinde** [comment]
> Will you also do the rebreather and medical gas mask?

check the picture for the rebreather

**Comment by Hyenh** [comment]
oh, I missed it

**Comment by snebl** [comment]
This is so much better than just hiding the snouts, also makes more sense.

File: Resources/Prototypes/_RMC14/Entities/Clothing/Masks/masks.yml [hunk 1]
```diff
@@ -68,6 +70,11 @@
     coverage: MOUTH
   - type: Item
     storedRotation: -90
+  - type: HideLayerClothing # TODO port pulled down sprite from SS13
+    slots:
+    - Hair
+    - HeadTop
+    - HeadSide
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Clothing/Masks/masks.yml:77]: Make this optional like helmets

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]
One note but also letting you know that the coif needs to be fixed as well
image
image

File: Resources/Prototypes/_RMC14/Entities/Clothing/Masks/masks.yml (hunk 1 above)
Comment (Dutch-VanDerLinde) [reply Resources/Prototypes/_RMC14/Entities/Clothing/Masks/masks.yml:77]: goes against parity cuz the coif hides hair if you have it equipped but if u really want it ill change it

Comment (Dutch-VanDerLinde) [reply Resources/Prototypes/_RMC14/Entities/Clothing/Masks/masks.yml:77]: actually ill just put it in a seperate PR

**Comment by Dutch-VanDerLinde** [comment]
Removed the coif hiding markings

**Review by QuietlyWhisper — APPROVED** [review]

**Review by DrSmugleaf — APPROVED** [review]
Thanks for the PR!

**Pull request merged**